xenit-alfresco/values.yaml

```diff
@@ -26,7 +26,9 @@ ingress:
 
 acs:
   replicas: 1
-  image: 'xenit/alfresco-repository-community:7.2.0'
+  registry: 'xenit'
```

Comment:
I think this should be docker.io and the repository is xenit/alfresco-repository-community , same for the other ones